.github/workflows/sanity.yml

```diff
@@ -0,0 +1,25 @@
+name: "Codeowners Validator"
+
+on:
+  workflow_dispatch:
+  push:
+  pull_request:
+    paths:
+    - '.github/CODEOWNERS'
+    - '.github/workflows/sanity.yaml'
+  schedule:
+    # Runs at 08:00 UTC every day
+    - cron:  '0 8 * * *'
+
+jobs:
+  sanity:
+    runs-on: ubuntu-latest
+    steps:
+      # Checks-out your repository, which is validated in the next step
+      - uses: actions/checkout@v2
+      - name: GitHub CODEOWNERS Validator
+        uses: mszostok/codeowners-validator@v0.5.0
+        with:
+          checks: "files,duppatterns,syntax"
```

Comment:
id be interested in seeing `notowned` added to the list here

Reply:
granted, its experimental.  
and to enable experimental check set EXPERIMENTAL_CHECKS=notowned environment variable.

Reply:
Yea, I had the `owners` enabled in iModel.js for a little while but it was failing due to a few people not being in the Org. I think we may hit the same issue here since a few of the people in the file aren't technically in the Org.

As for `notowned`, I could add that but I figured it's pretty much covered by `*` but if we get a more complicated config then it may make sense.